gm4_washing_tanks/pack.mcmeta

```diff
@@ -0,0 +1,32 @@
+{
+    "pack": {
+        "pack_format": 5,
+        "description": "A Gamemode 4 Module"
+    },
+    "module_name": "Washing Tanks",
+    "module_id": "washing_tanks",
+    "creation_meta":"Created using the Gamemode 4 Module Template Generator (gm4.co/modules/generator)",
+    "site_description": "Dye! Die!",
+    "site_categories": [
+	    "Liquid Tanks"
+    ],
+    "video_link": "",
+    "wiki_link": "wiki.gm4.co/wiki/Liquid_Tanks/Washing_Tanks",
+    "required_modules": [
+        "liquid_tanks"
+    ],
+    "credits": {
+        "Creator": [
+            [
+                "Bunnygamers",
+                "https://twitter.com/BunnygamersMC"
+            ]
+        ],
+	"With help from": [
+	    [
+	        "Misode",
+		"https://twitter.com/misoloo"
+	    ]
+	]
```

Comment:
This may look weird on the website, might wanna consider just adding misode to the creators? Additionally, misode's twitter link is outdated (misode_).